tests/recipient_validation/test_phone_number.py

```diff
@@ -61,8 +61,6 @@
     "0800 123 4567",  # freephone
     "0800 123 456",  # shorter freephone
     "0800 11 11",  # shortest freephone
-    "0845 46 46",  # short premium
-    "0900 123 4567",  # premium
```

Comment:
Not sure about the naming convention for these lists.

Premium numbers **are** valid, we just don't want them to pass validation.

Reply:
Put these into the same list and removed the extra test